### DISCUSSION TITLE
Is transfer learning possible?

### DISCUSSION THREAD

**Discussion by Ramnathan-zoimeet** [PaddlePaddle/PaddleSpeech#608] [Q&A]
So i understand that transfer learning with EN8K acoustic model is not possible with the develop branch of DeepSpeech2, although is there any other way to train on EN8K model or is there a possibility way access the baidu data source which could help in recreating this model from scratch?

**Comment by zh794390558** [comment]
I can not access EN8K dataset, if you want using this model , you can dump the 1.8 model and reload by 2.x. Then check if the model  forward has the same process.
